current/standard-derivatives.md

```diff
@@ -0,0 +1,552 @@
+    .orderBy(desc("md5"))
+    .write.format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/audio")
+
+// Images.
+warcs
+  .images()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"width", $"height", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/image")
+
+// PDFs.
+warcs
+  .pdfs()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/pdf")
+
+// Presentation Program Files.
+warcs
+  .presentationProgramFiles()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/presentation-program")
+
+// Spreadsheets.
+warcs
+  .spreadsheets()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/spreadsheet")
+
+// Text Files.
+warcs
+  .textFiles()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/text")
+
+// Videos.
+warcs
+  .videos()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/video")
+
+// Word Processor Files.
+warcs
+  .wordProcessorFiles()
+    .select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_type_tika", $"md5")
+    .orderBy(desc("md5"))
+    .write
+    .format("csv")
+    .option("header","true")
+    .mode("Overwrite")
+    .save("s3a://your-derivative-bucket/word-processor")
+
+sys.exit
+```
+
+### Python DF
+
+TODO
+
+## Extract Binary Info to Parquet
+
+How do I extract binary information of PDFs, audio files, video files, word processor files, spreadsheet files, presentation program files, and text files to the [Apache Parquet](https://parquet.apache.org/) format to [work with later](df-results.md#what-to-do-with-dataframe-results)?
+
+### Scala RDD
+
+TODO
+
+### Scala DF
+
+```scala
+import io.archivesunleashed._
+import io.archivesunleashed.df._
+
+// Web archive collection.
+val warcs = RecordLoader.loadArchives("/path/to/data", sc)
+
+// Web graph.
+warcs
```

Comment:
Same comment... instead of writing to {CSV, S3, Parquet}: organize the "meat" together, and just say the different write options.

I think that was kinda the goal with this:
https://github.com/archivesunleashed/aut-docs-new/blob/master/current/df-results.md

Maybe even just add to it? Since I think the write options are applicable everywhere?